stdlib/queue.gr

```diff
@@ -137,6 +137,37 @@ provide let pop = queue => {
   }
 }
 
+/**
+ * Converts a queue into a list of its elements.
+ *
+ * @param queue: The queue to convert
+ * @returns A list containing all queue values
+ *
+ * @since v0.6.0
+ */
+provide let toList = queue => {
+  let lst = List.init(queue.size, i => match (queue.array[queue.headIndex +
+    i]) {
```

Comment:
You'd have to mod this by the queue size since the elements in the queue could wrap around past the end and back to the beginning of the array

Reply:
so I would have todo `[(queue.headIndex + i) % queue.size`?

Reply:
It would be the length of `queue.array`. You can take a look at some of the other functions for an idea of what is done

Reply:
Made that change

Reply:
Needs a test where they'd wrap back around

Reply:
@spotandjake I'm not sure if this got lost but it still needs the test I mentioned.

Reply:
Added that test